worker/server_state.go

```diff
@@ -128,9 +128,8 @@ func (s *ServerState) initStorage() {
 		// Write Ahead Log directory
 		x.Checkf(os.MkdirAll(Config.WALDir, 0700), "Error while creating WAL dir.")
 		opt := badger.LSMOnlyOptions(Config.WALDir)
```

Comment:
We are relying on the default badger options for disabling. Is it okay? Or an explicit disable should be done?